stable/netsil/templates/collector.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  namespace: {{ .Chart.Name }}
+  name: collector
+  labels:
+    app: {{ template "name" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: collector
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{ .Chart.Name }}
+        component: collector
+    spec:
+      hostNetwork: true
```

Comment:
So I don't know if this is possible, but can we detect k8s server version and add `dnsPolicy: ClusterFirstWithHostNet` if version >= 1.6? Otherwise DNS won't work from the collectors (Well, there is a way for it to work, but we'd have to overwrite `resolv.conf` ourselves).